internal/core/application/blockchain_listener.go

```diff
@@ -171,6 +172,29 @@ func (b *blockchainListener) listenToEventChannel() {
 				log.Warnf("trying to confirm or add unspents: %v", err)
 				break
 			}
+
+			// Invoke webhooks registered for action TradeSettled
+			if webhookManager != nil {
```

Comment:
Method should not use external variable, webhookManager(maybe we should call all interfaces as Service, somewhere we use Manager somewhere Service) should be imported as a service(interface).